frontend/src/Search.elm

```diff
@@ -1024,16 +1067,7 @@ viewChannels outMsg selectedChannel =
                 )
             ]
         ]
-        (if List.member selectedChannel channels then
-            []
-
-         else
-            [ p [ class "alert alert-error" ]
-                [ h4 [] [ text "Wrong channel selected!" ]
-                , text <| "Please select one of the channels above!"
-                ]
-            ]
-        )
+        []
```

Comment:
Please remove the now useless `List.append` here (and while you're at it, the wrapping `[ div [] ... ]` as there's already one).